.github/workflows/ci.yml

```
@@ -0,0 +1,16 @@
+on:
+  push:
+    paths:
+      - 'apps/fda-gov-v2/**'
+  pull_request:
+    paths:
+      - 'apps/fda-gov-v2/**'
+
+jobs:
+  fda-gov-v2:
+    name: FDA Gov v2
+    uses: ./apps/fda-gov-v2/.github/workflows/fda-gov-v2.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./apps/fda-gov-v2/.github/workflows/fda-gov-v2.yml": open /home/jailuser/git/apps/fda-gov-v2/.github/workflows/fda-gov-v2.yml: no such file or directory

(workflow-call)